<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabViewPage</class>
 <widget class="QWidget" name="LabViewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LabView</string>
  </property>
  <widget class="QGroupBox" name="generalAmbBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Ambiance Générale</string>
   </property>
   <widget class="QComboBox" name="selectAmb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="colorBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Couleur Générale</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridColorSelect">
     <item row="2" column="2">
      <widget class="QSpinBox" name="bSpin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="g">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="g_label">
       <property name="text">
        <string>V :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="r_label">
       <property name="text">
        <string>R :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="b">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="r">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="rSpin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="b_label">
       <property name="text">
        <string>B :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="gSpin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="intensite_label">
       <property name="text">
        <string>Int. :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="intensite">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="intensiteSpin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="rgbLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>HHHHHH</string>
    </property>
   </widget>
   <widget class="QLabel" name="rgbLineEditLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Code RGB (#) :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showColor">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>170</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="lightBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>341</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Lumière blanche</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>184</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="selectLightType">
     <item>
      <widget class="QRadioButton" name="globalLightRadio">
       <property name="text">
        <string>Générale</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="individualLightRadio">
       <property name="text">
        <string>Individuelle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>311</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="intensityVLabel">
       <property name="text">
        <string>Verticale :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="3">
      <widget class="QSpinBox" name="intensityVSpin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="intensityV">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QLabel" name="intensityHLabel">
       <property name="text">
        <string>Horizontale :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="2">
      <widget class="QSpinBox" name="intensityHSpin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSlider" name="intensityH">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="lockButton">
    <property name="geometry">
     <rect>
      <x>323</x>
      <y>60</y>
      <width>25</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../ressources.qrc">
      <normaloff>:/img/img/lock-open.png</normaloff>
      <activeon>:/img/img/lock-close.png</activeon>:/img/img/lock-open.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="profilBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Profile</string>
   </property>
   <widget class="QPushButton" name="saveProfil">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
    <property name="icon">
     <iconset resource="../ressources.qrc">
      <normaloff>:/img/img/save.png</normaloff>:/img/img/save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="loadProfil">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger</string>
    </property>
    <property name="icon">
     <iconset resource="../ressources.qrc">
      <normaloff>:/img/img/upload.png</normaloff>:/img/img/upload.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="currentProfil">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Profile par defaut</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="selectFaceBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>361</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Selection des faces</string>
   </property>
   <widget class="QPushButton" name="backFace">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>125</width>
      <height>125</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/img/pattern_back.png);</string>
    </property>
    <property name="text">
     <string>Fond</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="topFace">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/img/pattern_t.png);</string>
    </property>
    <property name="text">
     <string>Haut</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="leftFace">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/img/pattern_l.png);</string>
    </property>
    <property name="text">
     <string>Gauche</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rightFace">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>80</y>
      <width>81</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/img/pattern_r.png);</string>
    </property>
    <property name="text">
     <string>Droite</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bottomFace">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>253</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/img/pattern_b.png);</string>
    </property>
    <property name="text">
     <string>Face Avant</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="frontFace">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/img/pattern_f.png);</string>
    </property>
    <property name="text">
     <string>Face avant</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="startVisu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>500</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualiser</string>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/img/img/play_green.png</normaloff>:/img/img/play_green.png</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="startRecording">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>500</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer la vidéo</string>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/img/img/record.png</normaloff>:/img/img/record.png</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cameraSelector">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrôle : </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="softwareRadio">
      <property name="text">
       <string>Logiciel</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="teleRadio">
      <property name="text">
       <string>Télécommande</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="startBroadcast">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>500</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Diffuser</string>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/img/img/broadcast.png</normaloff>:/img/img/broadcast.png</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="minimizeMaximize">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>21</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
